pulp_container/app/registry_api.py

```diff
@@ -86,8 +84,6 @@
     SIGNATURE_TYPE,
 )
 
-FakeView = namedtuple("FakeView", ["action", "get_object"])
-
```

Comment:
So all uses of this had moved elsewhere before?

Reply:
It seems like it was introduced in https://github.com/pulp/pulp_container/commit/8ca2f0e88462f78c76423cce8af838d8d3676e85, but it was never ever used.